Node/Managing Node-NPM versions.md

```diff
@@ -0,0 +1,126 @@
+
+**You should always aim to use latest LTS version available on your project**
+
+#### Maintenance phase
+
+Releases within this phase will receive critical bug fixes and security updates for 18 months in case of LTS(even-numbered) and 2 months in case of regular(odd-numbered) releases after entering this phase.
+
+### Ensuring same version of Node is being used on all environments
+
+It is very important to keep the same version of Node across the environments. When you update the version of Node in your project, you should take appropriate actions that the new version is also propagated/updated on build/production servers by notifying people responsible for those environments. By not running the same versions you could run into unexpected problems with failing builds, incorrect dependencies etc.
+
+### Managing versions
+
+There are couple of ways for managing versions of Node. This section lists couple of options and generally, all options are fine as long as you are proficient at installing, updating and switching Node versions using that method. These probably aren't all available version managers on the internet, but are probably most popular.
+
+#### n
+
+[n](https://github.com/tj/n) is both powerful and easy to use version manager for Node and should be preferred method for managing Node versions.
+You can create a `.n-node-version`, `.node-version` or `.nvmrc` file containing a Node version number. Running `n auto` will look for version specified in one of those files in that order and switch to it.
+
+```bash
+#Install n by running script
+npm install -g n
+
+#Install latest Node version
+n latest
+
+#Install specific version of Node by specifying version or use an alias
+n <version>
+n lts
+
+#Execute to check Node version currently in use and available versions
+n
+
+#Uninstall Node version(this does not affect cached versions)
+n uninstall <version>
+
+#Uninstall cached Node version
+n rm <version>
+```
+
+#### nvm - Node version manager
+
+[Node version manager](https://github.com/nvm-sh/nvm) is a version manager similar to `n` and a good alternative. You can create a `.nvmrc` file containing a Node version number. Running `use`, `install`, `which` etc. commands will use the version specified in the file if none was provided in the command line. When using `nvm` and installing global packages, those packages will only be global for that specific version. eg. You currently use Node 16 and install `polyglot-cli`, if you switch to v18, the package won't be available there until you install it.
+
+```bash
+#Install NVM by running script
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.2/install.sh | bash
```

Comment:
I would avoid writing installation instructions for any of the mentioned node version management tools, as these steps are likely to change in the future and we would have to update our handbook. You even have nvm version in this command. I think it will be sufficient to just link to the official website for the tool and instruct the reader to follow the instructions there.